examples/example/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conan import ConanFile, conan_version
+from conan.tools.cmake import CMake, cmake_layout
+
+class Example(ConanFile):
+
+    def set_name(self):
+        if self.name is None:
+            self.name = 'example'
+
+    def set_version(self):
+        if self.version is None:
+            self.version = '0.1.0'
+
+    license = 'ISC'
+    author = 'John Freeman <jfreeman08@gmail.com>'
+
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    options = {'shared': [True, False], 'fPIC': [True, False]}
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+        'xrpl:with_xrpld': False,
+    }
+
+    requires = ['xrpl/2.2.0-rc1@jfreeman/nodestore']
+    generators = ['CMakeDeps', 'CMakeToolchain']
+
+    exports_sources = [
```

Comment:
Is this needed in this `example` project ?

Reply:
No, it's just part of the template I use.